docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -300,35 +289,6 @@ The list of the available Docker images can be found on https://quay.io/reposito
 Be aware that a given Quarkus version might not be compatible with all the images available.
 ====
 
-Another way of customizing the native build image build process is to configure the task inside the Gradle build script. If for example it is required to set the `enableHttpUrlHandler`, it can be done like so:
-
-[source,groovy]
-----
-buildNative {
-    enableHttpUrlHandler = true
-}
-----
-
-or, if you use the Gradle Kotlin DSL:
-
-[source,kotlin,subs=attributes+]
-----
-tasks {
-    named<QuarkusNative>("buildNative") {
-        setEnableHttpUrlHandler(true)
-    }
-}
-----
-
-
```

Comment:
@geoand this PR removes the possibility of doing this, WDYT?

Reply:
Had we deprecated it before removing it?

Reply:
If not, I think we should first deprecate it (while also removing from the docs)

Reply:
I agree with @geoand 

@glefloch do you think it would be feasible if we had a PR deprecating this feature first?

Reply:
yes for sure, I rollback the code and keep the docs as this?

Reply:
I was thinking of keeping this PR as is and having another PR updating the docs saying that this feature is deprecated and will be removed in the next release

Reply:
Oh ok. I misunderstood. I updated the code. I will open an other pull request saying the task is deprecated.